plugins/scaffolder-backend/src/ScaffolderPlugin.ts

```diff
@@ -36,7 +36,6 @@ import { createRouter } from './service/router';
  */
 export type ScaffolderPluginOptions = {
   actions?: TemplateAction<any>[];
-  taskWorkers?: number;
```

Comment:
We need to keep this option as it would be a breaking change and require a major release if we removed it.
I think what we should do is default it to one and deprecate this option.

Reply:
I added this option back. Thanks!